index.bs

```diff
@@ -1302,6 +1302,12 @@ shared among all [=environment settings objects=].
 Note: This would ideally use <a spec=storage>storage bottles</a> to provide access to the attribution caches.
 However attribution data is inherently cross-site, and operations on storage would need to span across all storage bottle maps.
 
+An <dfn>internal ID</dfn> is an integer.
+
+To <dfn>get the next internal ID</dfn>, return an [=internal ID=] strictly
```

Comment:
IIUC no, but it's unclear because in the existing spec the phrase `unique string` doesn't specify the scope of the uniqueness. Maybe we can address that first, if we want.

That said, it may be beneficial later to use a single sequence to allow sources and reports to be ordered as well.

Reply:
Fine with me to leave as is. Thanks.